api/spec/requests/spree/api/v2/storefront/account/addresses_spec.rb

```diff
@@ -67,6 +67,23 @@
       end
     end
 
+    context 'when address from countries that are supported in the current store' do
+      let!(:country) { create(:country, name: 'France') }
+      let!(:zone) { create(:zone, name: 'EU_VAT', countries: [country], kind: 'country') }
+      let!(:eu_address) { create(:address, user_id: user.id, country: country) }
+
+      before do
+        Spree::Config[:checkout_zone] = zone.name
```

Comment:
@v10110 please do not use this preference, please use `current_store.checkout_zone` instead